src/tokens/RwaTokenFactory.t.sol

```diff
@@ -0,0 +1,88 @@
+/* prettier-disable */
+
+/**
+ * This file is a copy of https://goerli.etherscan.io/address/0xeb7C7DE82c3b05BD4059f11aE8f43dD7f1595bce#code.
+ * The only change is the solidity version, since this repo is using 0.6.x
+ */
+
+////// src/RwaToken.sol
```

Comment:
These comments should probably be removed and/or replaced.
Since it's a new file, it does not make sense to disable prettier, for example.